07_PDDE_Checker.R

```diff
@@ -227,9 +229,46 @@ es_no_tracking_method <- Project %>%
   ) %>%
   select(all_of(PDDEcols))
 
+
+# Zero Utilization --------------------------------------------------------
+
+projects_w_beds <- Inventory %>%
+  filter(
+    BedInventory > 0 &
+      coalesce(InventoryEndDate, meta_HUDCSV_Export_End) >= meta_HUDCSV_Export_Start &
```

Comment:
super small thing, but we do something similar when we check whether Inventory Dates are outside the operating dates. However, in that check we do as.Date(meta_HUDCSV_Export_End). Do we want to do the same thing here?

Reply:
Yes- good catch

Reply:
oh wait, I think it's meta_HUDCSV_ExportDate we do that for, since it comes in as ymd_hms